StripePayments/StripePayments/Source/Captcha/PassiveCaptcha.swift

```diff
@@ -50,7 +50,7 @@ private enum CaptchaResult {
     }
 
     private let passiveCaptcha: PassiveCaptcha
-    private var validationTask: Task<String?, Never>?
+    private var validationTask: Task<CaptchaResult?, Never>?
```

Comment:
I think this might be more idiomatically a `Task<String, Swift.Error>` but let's figure that out in a follow-up PR.